src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/ManageApprentices/Index.cshtml

```diff
@@ -59,6 +59,14 @@ else
             }
             else
             {
+                <div id="total-count">
+                    <p>
+                        <strong>@Model.TotalResults</strong> record@(Model.TotalResults > 1 ? "s" : null) found.
+                    </p>
+                    <p>
+                        You can search again or <a href="@Url.Action("Index")" title="see all apprentices" class="text-link">see all apprentices</a>
```

Comment:
Same comments as on employer version:

This may be as specified in the story, but I think that inviting the user to "search again" and reset back to all apprenticeships (the second para) should not be shown if the user didn't submit a search or filter. The invitation to search "again" is meaningless, and the reset link doesn't do anything (well, it resets the page number to 1 so a bit confusing).